web_tree_no_open/README.rst

```diff
@@ -0,0 +1,81 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Web Tree No Open
```

Comment:
Hi @codingforfun thanks for your contribution.
Please add a line before the document title.